workflow.swift

```diff
@@ -1,25 +1,69 @@
-
+import assert;
 import io;
 import launch;
 import string;
+import sys;
 
-program1 = "./heat_transfer_adios2";
-arguments1 = split("heat  4 3  40 50  6 500", " ");
-printf("swift: launching: %s", program1);
-exit_code1 = @par=12 launch(program1, arguments1);
-printf("swift: received exit code: %d", exit_code1);
-if (exit_code1 != 0)
-{
-  printf("swift: The launched application did not succeed.");
+string rmethod;
+void ready;
+int availproc;
+
+int htproc_x = 4;
+int htproc_y = 3;
+int htproc = htproc_x * htproc_y;
+int swproc = 3;
+int dsproc = 1;
+
+availproc = turbine_workers();
+rmethod = argv("s", "FLEXPATH");
```

Comment:
@kshitij-v-mehta The -s flag is being passed as an argument to workflow.tic, rather than as a turbine option. That allows run-workflow.sh to pass on the transport type. I had thought I had picked something that didn't stomp on any turbine flags, but apparantly not 😬. Perhaps a different letter would be less confusing? -t for transport maybe?